dash_bio/component_factory/_clustergram.py

```diff
@@ -50,15 +50,14 @@ def Clustergram(
 Keyword arguments:
 
 - data (ndarray; required): Matrix of observations as array of arrays
-- generate_curves_dict (bool; optional): Whether or not to return a
+- generate_curves_dict (bool; default false): Whether or not to return a
     dictionary containing information about the cluster number
     associated with each curve number in the graph. (May be useful
     if one wishes to capture the cluster number that is clicked.)
-    (Default: False)
-- return_computed_traces (bool; optional): Whether or not to return
+- return_computed_traces (bool; default false): Whether or not to return
```

Comment:
```suggestion
- return_computed_traces (bool; default False): Whether or not to return
```

Reply:
These are meant to mimic the auto-generated docstrings as closely as possible. "True" and "False" are not capitalized in JavaScript, and they don't end up capitalized in the generated Python docstrings either.

Reply:
I'll commit these suggestions and follow up in the Dash repo that is used to generate Python files so that the React components match!

Reply:
> These are meant to mimic the auto-generated docstrings as closely as possible. "True" and "False" are not capitalized in JavaScript, and they don't end up capitalized in the generated Python docstrings either.

I respect the logic, but I think it's better to make docstrings consistent the other way around, the way you suggest:

> I'll commit these suggestions and follow up in the Dash repo that is used to generate Python files so that the React components match!

When writing pure Python components, i.e., writing directly in Python, I don't think we want to mimic their 'inheriting' from JS, just because React components happen to be converted too naively for the time being.